ui/style.css

```diff
@@ -5095,6 +5109,126 @@ span.tag-reports-input[contenteditable]:empty::before {
     user-select: none;
 }
 
+.leaderboard-officer-title {
+    background-color: var(--color-4);
+    color: white;
+    font-size: 18px;
+    font-weight: bolder;
+    padding-top: 5px;
+    padding-bottom: 5px;
+    text-align: center;
+    margin: auto;
+    margin-top: 10px;
+    margin-bottom: 0px;
+    width: 95%;
+    align-items: center;
+    user-select: none;
+}
+
+.leaderboard-title:hover {
+    background-color: var(--color-3);
+}
+
+.leaderboard-search-input {
+    background-color: var(--color-3);
+    color: white;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    text-align: center;
+    margin: auto;
+    margin-top: 10px;
+    margin-bottom: 0px;
+    width: 95%;
+    align-items: center;
+    user-select: none;
+    display: none;
+}
+
+#leaderboard-search-input {
+    outline: none;
+    border: none;
+    width: 95%;
+    color: white;
+    font-size: 15px;
+    background-color: rgba(0, 0, 0, 0);
+    margin: auto;
+}
+
+#leaderboard-search-input::placeholder {
+    color: white;
+}
+
+.leaderboard-items {
+    display: flex;
+    flex-direction: column;
+    overflow: auto;
+    margin: auto;
+    margin-top: 15px;

```

Comment:
Combine the three margin properties to `margin: 15px auto 0 auto;`

Reply:
Did that thank you <3